docs/src/main/sphinx/release/release-397.md

```diff
@@ -0,0 +1,51 @@
+# Release 397 (21 Sep 2022)
+
+## General
+
+* Fix incorrect parsing of invalid values in cast from `varchar` to `timestamp`. ({issue}`14164`)
+* Fix potential incorrect results for queries where all data scanned from tables
+  was processed in the leaf stage and was not passed downstream. ({issue}`14168`)
+* Fix `EXPLAIN (TYPE IO)` query failure for certain queries with redundant table
+  scans. ({issue}`10398`)
+
+## JDBC driver
+
+* Fix potential memory leak when cancelling statements. ({issue}`14176`)
+
+## Delta Lake connector
+
+* Add `parquet.optimized-writer.enabled` as the new configuration property and
+  `parquet_optimized_writer_enabled` as the new session property for enabling
+  the optimized parquet writer. Deprecate the
+  `parquet.experimental-optimized-writer.enabled` configuration property and
+  `experimental_parquet_optimized_writer_enabled` session property. ({issue}`14137`)
+
+## Hive connector
+
+* Add `parquet.optimized-writer.enabled` as the new configuration property and
+  `parquet_optimized_writer_enabled` as the new session property for enabling
+  the optimized parquet writer. Deprecate the
+  `parquet.experimental-optimized-writer.enabled` configuration property and
+  `experimental_parquet_optimized_writer_enabled` session property. ({issue}`14137`)
+* Add support for Hive splits for JSON inputs in S3 Select. ({issue}`14040`)
```

Comment:
What does this mean? What's the impact or user-visible effect of this change?

From the description in the issue:

> Trino client will return results faster when S3 Select pushdown is enabled for uncompressed JSON files:
set SESSION hive.s3_select_pushdown_enabled=true;

So I would just say:

> Improve performance when querying JSON data and Hive S3 Select pushdown is enabled.